tests/ui/command/command-current-dir.rs

```diff
@@ -1,4 +1,5 @@
 //@ run-pass
+//@ no-prefer-dynamic We move the binary around, so do not depend dynamically on libstd
```

Comment:
Why is this changed needed? Was it linking to the compiler's std copy before? If so, why?

Reply:
The test was buggy. It compiled a Rust binary that dynamically linked to libstd.so. Then it moved the binary around, which should have broken it. However, the binary somehow linked to the compiler's std copy instead (!), so it worked by accident.

Reply:
Since UI tests are built with `-Cprefer-dynamic` did they all (modulo `no-prefer-dynamic`) link to the std?

Reply:
I don't think so, no other tests broke. It was probably just caused by the fact that this binary was moved around, lost its link to the target library and then somehow got dynamically linked to the compiler one instead.